src/styles/main.scss

```diff
@@ -0,0 +1,17 @@
+@import "./utils/variables";
+@import "./utils/mixins.scss";
+
+@import "./reset";
+
+@import "./block/card";
+@import "./block/stars";
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Roboto", Arial, sans-serif;
+  background-color: $color-white;
+  margin: 0;
+}
```

Comment:
Move general styles and reset to a separate file to keep your main clean and clear